ament_cmake_core/cmake/environment_hooks/ament_cmake_environment_hooks_package_hook.cmake

```diff
@@ -1,6 +1,11 @@
+if(WIN32)
+    set(_ext .bat)
```

Comment:
I would recommend quoting the value for better readability. Same two lines below and in `ament_cmake_export_libraries-extras.cmake`.